misk/src/main/kotlin/misk/security/authz/AccessControlModule.kt

```diff
@@ -10,5 +10,6 @@ class AccessControlModule : ActionScopedProviderModule() {
   override fun configureProviders() {
     multibind<ApplicationInterceptor.Factory>().to<AccessInterceptor.Factory>()
     newMultibinder<AccessAnnotationEntry>()
+    newMultibinder<String, ExcludeServiceFromWildcards>()
```

Comment:
I wonder if this is the cleanest way

Alternatives would be... an `excludes` in the `AllowAll` annotation?

Reply:
I wanted other modules (or the installing module) to easily add/set entries.